src/components/parent-form/ParticipantCreatedPage.tsx

```diff
@@ -112,9 +114,15 @@ export const ParticipantCreatedPage: React.FC<ParticipantCreatedPageProps> = ({
                                 : "Error: Participant not added successfully"}
                         </Text>
                         <Text maxW={400} textAlign="center" pt={3} pb={9}>
-                            {successful === "success"
-                                ? "Participant has been successfully added as a participant in your account"
-                                : "There was an error adding the participant. Please contact us"}
+                            {successful === "success" ? (
+                                <div>
+                                    <strong>{name}</strong> has been
+                                    successfully added as a participant in your
+                                    account!
+                                </div>
```

Comment:
We want to minimize the amount of non-chakra components, especially div.
```suggestion
                                    <strong>{name}</strong> has been
                                    successfully added as a participant in your
                                    account!
```

^ if that doesn't work, maybe just use empty tags instead, "<>"